enterprise/frontend/src/metabase-enterprise/api/tenants.ts

```diff
@@ -0,0 +1,55 @@
+import type {
+  CreateTenantInput,
+  Tenant,
+  UpdateTenantInput,
+} from "metabase-types/api";
+
+import { EnterpriseApi } from "./api";
+import { idTag, invalidateTags, listTag } from "./tags";
+
+export const tenantsApi = EnterpriseApi.injectEndpoints({
+  endpoints: (builder) => ({
+    createTenant: builder.mutation<void, CreateTenantInput>({
+      query: (body) => ({
+        method: "POST",
+        url: "/api/ee/tenants",
```

Comment:
don't we usually use singular nouns for API endpoints? `/api/ee/tenant` instead of `/api/ee/tenants` ?